src/inspectors/package/RustPackageInspector.ts

```diff
@@ -0,0 +1,415 @@
+import { PackageInspectorBase } from './PackageInspectorBase';
+import { injectable, inject } from 'inversify';
+import { Types } from '../../types';
+import { IFileInspector } from '../IFileInspector';
+
+import * as TOML from '@iarna/toml';
+import { DependencyType, Package, PackageVersion } from '../IPackageInspector';
+
+const isRecord = function (v: unknown): v is Record<string, unknown> {
+  return typeof v === 'object' && v !== null;
+};
+const isString = function (v: unknown): v is string {
+  return typeof v === 'string';
+};
+const isOptString = function (v: unknown): v is string | undefined {
+  return typeof v === 'string' || v === undefined;
+};
+const isOptBool = function (v: unknown): v is boolean | undefined {
+  return typeof v === 'boolean' || v === undefined;
+};
+const isOptStringArray = function (v: unknown): v is string[] | undefined {
+  return v === undefined || (Array.isArray(v) && v.every((e) => typeof e === 'string'));
+};
+
+/**
+ * Similar to `lodash.conformsTo` but runs the check on `undefined` instead of failing.
+ */
+const conformsOptional = function (check: Record<string, (v: unknown) => boolean>, value: Record<string, unknown>): boolean {
```

Comment:
Is it possible at all to add this to the linter? It did warn me about `function <name>()` and told me to use `const <name> = function ()` but I'd appreciate if it could then tell me to use arrow functions as well.

Reply:
I'm not sure actually. I can check it.